zetta_utils/layer/volumetric/backend.py

```diff
@@ -19,3 +22,35 @@ def enforce_chunk_aligned_writes(self) -> bool:
     @abstractmethod
     def enforce_chunk_aligned_writes(self, value: bool) -> None:
         ...
+
+    @abstractmethod
+    def get_voxel_offset(self, resolution: Vec3D) -> IntVec3D:
+        ...
+
+    @abstractmethod
+    def set_voxel_offset(self, voxel_offset: IntVec3D, resolution: Vec3D) -> None:
+        ...
+
+    """ Sets the voxel offset at the given resolution for the backend.
```

Comment:
This comment should go instead of `...`?

Reply:
Oops.